### PR TITLE
Populated data base with planets and characters

### PR DESCRIPTION
**Pull request by saroskovic**

### CODE REVIEW

File: src/main/java/axiomq/com/starwars/controllers/UserController.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import axiomq.com.starwars.services.UserService;
+import lombok.AllArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@AllArgsConstructor
+@RequestMapping("/user")
+public class UserController {
```
Comment (stefanbesovic) [inline src/main/java/axiomq/com/starwars/controllers/UserController.java:13]: This should be in another pull request. You are mixing implementations. This is not included in commit message and does not have any connections in populating Character and Planet tables.

Comment (stefanbesovic) [reply src/main/java/axiomq/com/starwars/controllers/UserController.java:13]: As well as everything that is connected to User (User.java, UserServiceImpl.java)

File: src/main/java/axiomq/com/starwars/services/implementations/PlanetServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+@RequiredArgsConstructor
+public class PlanetServiceImpl implements PlanetService {
+
+    private final PlanetRepository planetRepository;
+
+    private final RestTemplate restTemplate;
+
+    private final PlanetConverter planetConverter;
+
+    String url = "https://swapi.dev/api/planets";
```
Comment (stefanbesovic) [inline src/main/java/axiomq/com/starwars/services/implementations/PlanetServiceImpl.java:25]: Do what Branimir told me, externalize this to application.properties, or some other property file.

File: src/main/java/axiomq/com/starwars/services/implementations/FilmServiceImpl.java [hunk 3]
```diff
@@ -1,21 +1,29 @@
...
 @RequiredArgsConstructor
 public class FilmServiceImpl implements FilmService {
 
+    private final RestTemplate restTemplate;
+
+    private final FilmConverter filmConverter;
+
     private final FilmRepository filmRepository;
 
+    String url = "https://swapi.dev/api/films";
```
Comment (stefanbesovic) [inline src/main/java/axiomq/com/starwars/services/implementations/FilmServiceImpl.java:25]: Externalize this also.

File: src/main/java/axiomq/com/starwars/controllers/FilmController.java [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+import java.util.List;
+
+@RestController
+@AllArgsConstructor
+@RequestMapping("/film")
+public class FilmController {
+
+    private FilmService filmService;
+
+    @GetMapping("/populatedb")
```
Comment (branimir-vujicic) [inline src/main/java/axiomq/com/starwars/controllers/FilmController.java:17]: move to DatabaseController

File: src/main/resources/data.sql [hunk 5]
```diff
@@ -0,0 +1 @@
+INSERT INTO user (email, password, user_type, username) VALUES ('zoran@mail.com', 'pass123', 'ADMIN' , 'zoran');
```
Comment (branimir-vujicic) [inline src/main/resources/data.sql:1]: move to test/resources
